### DISCUSSION TITLE
More about Material Design

### DISCUSSION THREAD

**Discussion by gaedlld** [squidfunk/mkdocs-material#2345] [General]
Hello,

At first, very good job ! And I have a question about it. 

Where does your material components come from ?
I mean, did you developed all design (components shape & animation) from zero or did you use a material component provider ?

I really like your nav-bar & search component, that I would like to use for a personal project.

Thanks,
Gaëtan

**Comment by squidfunk** [comment]
All components and CSS is written from scratch, no frameworks were used 😉
